bin/Phantom/cards/examples/phantomProduction_example.cfg

```diff
@@ -0,0 +1,177 @@
+# ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ---- #
+#                                                                                        #
+#           template cfg file for the usage of the phantom generator in CMS              #
+#                                                                                        #
+# see https://twiki.cern.ch/twiki/bin/view/CMS/PhantomGeneratorCMS for more details      #
+#                                                                                        #
+# ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ---- #
+
+
+[general]
+# where the zipped version of the compiled phantom code can be found
+# and where the gridpacks will be created
+
+# CMSSW environment for the gridpack creation
+CMSSW = CMSSW_9_2_6_patch1
```

Comment:
suggest to use CMSSW_9_2_11 as we just did for POWHEG (cfr https://twiki.cern.ch/twiki/bin/viewauth/CMS/PowhegBOXPrecompiled#Download_all_the_necessary_scrip)

Reply:
Ok, that's not a problem, this is just the example for testing matters. Whatever is the release I will use for the test in genproduction (and then for the actual production) I will put it there

Reply:
I suggest to keep the CMSSW version "fixed" once for all and use the trick in runcmsgrid_phantom.sh similarly to [POWHEG](https://github.com/cms-sw/genproductions/blob/master/bin/Powheg/runcmsgrid_powheg.sh#L31-L60) to override the CMSSW version in production. This way you can use the same compiled version for 2016 and 2017 MC productions, for instance

Reply:
ok, I can add to the cfg file that it has not to be changed. The script is already compliant with that, for it sources the CMSSW as indicated in the config file before generating the events.